documentation/pom.xml

```diff
@@ -180,6 +180,7 @@
                                 <jdkdocroot>${jdk.docroot}</jdkdocroot>
                                 <javaeedocroot>${javaee.docroot}</javaeedocroot>
                                 <download_url>${download.url}</download_url>
+                                <schemaVersion>${parsedVersion.majorVersion}.${parsedVersion.minorVersion}</schemaVersion>
```

Comment:
@ryanemerson snap! I was just looking to add a variable for the schema version. need it for server config examples. this looks good to me but schemaVersion also needs to go into documentation/src/main/asciidoc/topics/attributes/product-attributes.adoc 

if you want I could add that separately in a PR I've got going now. but just saying the product docs don't use the pom.xml :(

Reply:
@oraNod So it's not possible (AFAIK) to use the `asciidoctor-maven-plugin` attributes as they are only seem to be applied to the .adoc files. Instead, I have performed the filtering on the `.xml` files using the `maven-resources-plugin` and then made it so that the `asciidoctor-maven-plugin` is executed using the content in `target/` which has the transformed xml.